php_phongo.c

```diff
@@ -2228,60 +2259,60 @@ static mongoc_client_t *php_phongo_make_mongo_client(const mongoc_uri_t *uri TSR
 	bson_version = "bundled";
 #endif
 
-	MONGOC_DEBUG("Creating Manager, phongo-%s[%s] - mongoc-%s(%s), libbson-%s(%s), php-%s",
+	MONGOC_DEBUG(
+		"Creating Manager, phongo-%s[%s] - mongoc-%s(%s), libbson-%s(%s), php-%s",
 		PHP_MONGODB_VERSION,
 		PHP_MONGODB_STABILITY,
 		MONGOC_VERSION_S,
 		mongoc_version,
 		BSON_VERSION_S,
 		bson_version,
-		PHP_VERSION
-	);
+		PHP_VERSION);
```

Comment:
I noticed that libmongoc would format this by indenting all subsequent lines at the next column after `MONGOC_DEBUG(`. I assume that won't work well because we use tabs instead of spaces for indents (it'd require mixing both for a leading indent).

Would you prefer `);` on its own line as in the original, or is this something we have no control over (like multi-line `if` conditions)?